page/sass/footer.scss

```diff
@@ -7,6 +7,10 @@
   color: $gray-between-light-and-dark;
   font-size: $font-size-normal-smaller;
 
+  &.hidden {
```

Comment:
hidden is bootstrap build in https://github.com/twbs/bootstrap-sass/blob/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/stylesheets/bootstrap/_utilities.scss#L45